### ISSUE TITLE
Adjust default save location screenshots

### ISSUE THREAD

**Issue by thesaigoneer** [nwg-piotr/nwg-panel#357]
**Is your feature request related to a problem? Please describe.**
When using the integrated screenshot utility screenshots are now saved in /home/'user'/Screenshots

**Describe the solution you'd like**
Instead of creating a new folder in the user's home directory I'd think it would be better to save in /home/'user'/Pictures/screenshots

**Additional context**
See the attached scrot for the current setup
![image](https://github.com/user-attachments/assets/2f5c6895-4889-4891-889a-a88c3e9b8692)

**Comment by nwg-piotr** [comment]
Yes, it's just a stupid habit of mine. I'll try to get it sorted out over the weekend. I've had no time for FOSS at all for the last two days (!!!).

**Comment by thesaigoneer** [comment]
No worries, mate! It is just sth very small i came across when making scrots. First of all: get some time off and a beer or 4. This is absolutely not urgent ;-)

**Comment by nwg-piotr** [comment]
That is the plan!

**Comment by nwg-piotr** [comment]
This will need a change in the `/usr/bin/screenshot` script, which is a part of the nwg-shell package. It'll have to wait for the nearest package update.

**Comment by thesaigoneer** [comment]
Thnx for looking into it, we will see it in the next stable release!
